fileSystem/views.py

```diff
@@ -174,24 +182,26 @@ def check_permissions(self, request):
     def on_completion(self, uploaded_file, request):
         name = uploaded_file.name
         path = settings.MEDIA_ROOT + name
-
-        f = file.objects.create(name=name, upload=path)
+        SaveFile(path, uploaded_file)
```

Comment:
This line save file to the our server storage. Do we want to do it?

Reply:
yes, because I send file from disk. It's simple than send by chunks to google.